src/amber/router/context.cr

```diff
@@ -11,9 +11,11 @@ class HTTP::Server::Context
 
   property route : Radix::Result(Amber::Route)
   getter router : Amber::Router::Router
+  getter cookies : Amber::Router::Cookies::Store
 
   def initialize(@request : HTTP::Request, @response : HTTP::Server::Response)
     @router = Amber::Router::Router.instance
+    @cookies = Amber::Router::Cookies::Store.build(request, Amber::Server.key_generator)
```

Comment:
It is nillable  and the `Amber::Router::Cookies::Store.build` only sets cookies if the Server.cookies_enabled? returns true